examples/with-graphql-hasura/app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "nextjs-hasura-example",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "test": "NODE_OPTIONS='--inspect' jest --watch --runInBand --testPathIgnorePatterns=[]"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.3.6",
```

Comment:
Let's use a more slim GraphQL client like `graphql-request`.

Reply:
`graphql-request` currently [doesn't support graphql subscriptions](https://github.com/prisma-labs/graphql-request/issues/77). I decided to use Apollo because its more feature complete, and felt the community would be better off starting off from that. What do you think?

Reply:
Replaced with `graphql-request`